boards/arm/blackpill_f401ce/support/openocd.cfg

```diff
@@ -0,0 +1,12 @@
+source [find board/stm32f4discovery.cfg]
```

Comment:
Even though it likely works, the cfg is for another board, so unless this board is supported by openocd better use something like

```
source [find interface/stlink.cfg]

transport select hla_swd

source [find target/stm32f4x.cfg]

reset_config srst_only
```